your_extension.info.yml

```diff
@@ -1,5 +1,7 @@
 name: Drupal extension scaffold
 type: module
+#type: theme
```

Comment:
please use sed in init.sh to replace `type: module` with `type: theme`. Do not add these commented lines.